libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -401,7 +402,7 @@ class MapboxNavigation @VisibleForTesting internal constructor(
      * Describes whether this instance of `MapboxNavigation` has been destroyed by calling
      * [onDestroy]. Once an instance is destroyed, it cannot be used anymore.
      *
-     * @see [MapboxNavigationProvider]
+     * @see [MapboxNavigationApp]
```

Comment:
You should also update the `IllegalStateException` message in the line 419.
And the docs for `requestRoadGraphDataUpdate` (line 845).

Reply:
Thanks for catching!